server/server.go

```diff
@@ -276,14 +276,16 @@ func (r *rodeServer) ValidatePolicy(ctx context.Context, policy *pb.ValidatePoli
 	mod, err := ast.ParseModule("validate_module", string(policy.Policy))
 	if err != nil {
 		log.Debug("failed to parse the policy", zap.Any("policy", err))
-		return &pb.ValidatePolicyResponse{
+		message := &pb.ValidatePolicyResponse{
 			Policy:  policy.Policy,
 			Compile: false,
 			Errors:  []string{err.Error()},
-		}, nil
+		}
+		s, _ := status.New(codes.InvalidArgument, "failed to parse the policy").WithDetails(message)
```

Comment:
Could this message match the others? `failed to compile the policy`

Reply:
Actually I'm looking at this more, and I'm not sure what's appropriate here. What's the difference between a policy failing to compile and a policy failing to parse?

Reply:
I can make them match. I guess it's semantics.

Failed to parse means it failed to generate and ast. Symbols that don't exist, no matching parentheses, etc

Failed to compile means that your code is correct but you didn't declare variables, setting stings to ints...

Reply:
Ohhh gotcha gotcha, thanks! You can keep them separate if you want, I can handle the difference in the UI just fine. I was under the impression they were the same sort of thing, but I could see value in returning two different messages for people who might be writing policy on their own.